doc/Type/DateTime.pod6

```diff
@@ -252,6 +252,51 @@ Returns an L<Instant|/type/Instant> object based on the invocant.
 
     say DateTime.new('2015-12-24T12:23:00+0200').Instant; # OUTPUT: «Instant:1450952616␤»
 
+=head2 method day-fraction
+
+Defined as:
+
+    method day-fraction(DateTime:D: --> Real:D)
+
+Returns the instant's time as a fraction of a 24-hour day.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').day-fraction; # OUTPUT: «0.5159772␤»
+
+Notice the C<day-fraction> value is the same as the fractional part of
+the C<modified-julian-date> for the same instant.
+
+=head2 method julian-date
+
+Defined as:
+
+    method julian-date(DateTime:D: --> Real:D)
+
+Returns the L<Julian Date|https://en.wikipedia.org/wiki/Julian_day> (JD) for the UTC date and time.
+
+    say DateTime.new('2021-12-24T12:23:00.43Z').julian-date; # OUTPUT: «2459573.0159772␤»
+
+The C<julian-date> starts at zero at the epoch starting at noon UTC on
+I<Monday, January 1, 4713 BC>, on the Gregorian calendar (the calendar
+in use in much of the world and in international commerce and travel)
+and is used in astronomy to define celestial objects' transits at the
```

Comment:
I'm grasping here for a word that might be better than *define*. *Determine*? *Denominate*?

Reply:
If it can be of help, Wikipedia defines the [Julian Day](https://en.wikipedia.org/wiki/Julian_day) as:

> The Julian day is the continuous count of days since the beginning of the Julian period, and is used primarily by astronomers, and in software for easily calculating elapsed days between two events

So I guess _calculate_ or _compute_ would be fine.

Reply:
I will change to calculate.